service/init.go

```diff
@@ -168,6 +169,14 @@ func (udr *UDR) FilterCli(c *cli.Context) (args []string) {
 	return args
 }
 
+func mongoConnect(url string, dbname string) error {
+	var mClient, errConnect = mongoapi.NewMongoClient(url, dbname)
```

Comment:
This API is timing out with 10 sec..so what we achieve by keeping 10 sec timer here as well ? Need some adjustment right?

Reply:
I increased the timeout to 3 mins by trying to connect for each 2 secs.  Is that reasonable ?